src/Template/Bake/Template/index.ctp

```diff
@@ -39,20 +39,21 @@ $fields = collection($fields)
     endforeach;
 %>
     </ul>
-</div>
-<div class="<%= $pluralVar %> index large-10 medium-9 columns">
+</nav>
+<div class="<%= $pluralVar %> index large-9 medium-8 columns" id="content">
```

Comment:
I completely disagree that class names need to be semantic. From working on a few large applications, 'semantic' class names create css sprawl. While grid classes look ugly they offer far simpler mechanics for code reuse and I've found them much easier to maintain in the context of larger applications.